cryptarchia/cryptarchia.py

```diff
@@ -592,6 +614,17 @@ def _infer_total_active_stake(
         )
         return int(prev_epoch.inferred_total_active_stake - h * blocks_per_slot_err)
 
+    def blocks_by_slot(self, from_slot: Slot) -> Generator[BlockHeader, None, None]:
+        # Returns blocks in the given range of slots in order of slot
+        # NOTE: In real implementation, this should be done by optimized data structures.
```

Comment:
I just realized that this can be done by iterating the `local_chain` and all `forks` backward. I'll update this function soon.